### PR TITLE
Document preferred attributions

### PR DESCRIPTION
**Pull request by MarkusObendrauf**
### Summary of changes

Add documentation for the new preferred attributions feature.

### Context and reason for change

This feature was added recently and should be documented

### How can the changes be tested

See the [updated USER_GIUIDE.md](https://github.com/MarkusObendrauf/OpossumUI/blob/document-preferred/USER_GUIDE.md#preferred_attributions)

### CODE REVIEW

File: USER_GUIDE.md [hunk 1]
```diff
@@ -365,3 +366,12 @@ filters that allows for filtering for attributions marked for follow-up, first p
...
 Clicking on the _edit_ buttons in the _name_ columns, navigates to the respective attribution in the `attribution view`.
+
+
+### Preferred Attributions <a name="preferred_attributions"></a>
+
+In the audit view, an attributions can be marked as preferred, to indicate that it is preferred over the displayed signals. This feature does not have any significance in OpossumUI; instead, it is intended to give additional information to tools that consume .opossum files. A preferred attribution will store origin IDs of signals visible to the user when it was marked as preferred.
+
+Only signals with a source marked as `isRelevantForPreference` can be preferred over. If no signal source has this flag set, then the feature is disabled.
+
+To mark an attribution as preferred, choose an attribution in the audit view, and open the ... context menu. From there, you can mark or unmark an attribution as preferred. When an attribution is marked as preferred, `preferred = true` is written to the .opossum file, and the origin IDs of all visible signals relevant for preference are written in the field `preferredOverOriginIds`. Preferred attributions are displayed with a star icon.
```
Comment (FlorianSchepers) [inline USER_GUIDE.md:377]: ... -> `...`

File: USER_GUIDE.md [hunk 2]
```diff
@@ -365,3 +366,12 @@ filters that allows for filtering for attributions marked for follow-up, first p
 are mutually exclusive.
 
 Clicking on the _edit_ buttons in the _name_ columns, navigates to the respective attribution in the `attribution view`.
+
+
+### Preferred Attributions <a name="preferred_attributions"></a>
+
+In the audit view, an attributions can be marked as preferred, to indicate that it is preferred over the displayed signals. This feature does not have any significance in OpossumUI; instead, it is intended to give additional information to tools that consume .opossum files. A preferred attribution will store origin IDs of signals visible to the user when it was marked as preferred.
```
Comment (FlorianSchepers) [inline USER_GUIDE.md:373]: Optional: "This feature does not have any immediate effect on the signals displayed in OpossumUI;"

**Review by FlorianSchepers — APPROVED** [review]
lgmt

File: USER_GUIDE.md (hunk 2 above)
Comment (MarkusObendrauf) [reply USER_GUIDE.md:373]: yeah looking back i dont like my wording. changed.

**Pull request merged**